dali/pipeline/operators/crop/crop.cc

```diff
@@ -29,20 +37,20 @@ DALI_SCHEMA(Crop)
     .AddOptionalArg("crop_pos_y",
                     R"code(Vertical position of the crop in image coordinates (0.0 - 1.0))code",
                     0.5f, true)
-    .AddOptionalArg("image_type",
-                    R"code(The color space of input and output image)code", DALI_RGB, false)
     .AddArg("crop",
             R"code(Size of the cropped image. If only a single value `c` is provided,
  the resulting crop will be square with size `(c,c)`)code", DALI_INT_VEC)
+    .AddParent("CastPermute")
```

Comment:
That means that crop by default inherits from Castpermute, even in those cases where Crop is used standalone. I think it is not advisable to merge those attribute contexts into a larger one

Reply:
*No, it does not!* It only means that "Crop" uses `image_type, output_dtype, output_layout` and their types and default values are defined in "CastPermute" schema. The implementation of corresponding operators could be quote different from the description of schemas.